Documentation/prerequisites-for-building.md

```diff
@@ -20,9 +20,11 @@ sudo apt-get update
 ```
 
 ```sh
-sudo apt-get install cmake clang-3.9 libicu52 libunwind8 uuid-dev libcurl4-openssl-dev zlib1g-dev libkrb5-dev
```

Comment:
There needs to be some ICU installed to make things work. It probably just happens that there is some ICU installed in the default Ubuntu. Just pointing it out...

Reply:
Seems like `libicu55` is installed by default. I've updated the `apt-get` with that.